modules/navier_stokes/test/tests/finite_volume/ins/turbulence/lid-driven/lid-driven-turb-capped.i

```diff
@@ -28,13 +28,11 @@ sigma_eps = 1.3
 C1_eps = 1.44
 C2_eps = 1.92
 C_mu = 0.09
+C_pl = 0.1
```

Comment:
Why 0.1 and not 10.0?

Reply:
If this is just a test to significantly cap the production, please disregard this comment

Reply:
this is a test to cap the production